.github/workflows/tests-mariadb-linux.yml

```
@@ -43,47 +43,30 @@ jobs:
         with:
           mariadb-version: ${{ inputs.mariadb }}
       - name: Set up Python ${{ matrix.python-version }}
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)